bump-version.php

```diff
@@ -0,0 +1,197 @@
+<?php
```

Comment:
It may make sense to put this file in `bin/` since it's a dev script and will already be ignored from the export by the current `.gitattribute` config. I don't have a strong opinion on it though.